source/conf.py

```diff
@@ -106,6 +106,7 @@
 # If true, keep warnings as "system message" paragraphs in the built documents.
 #keep_warnings = False
 
+intersphinx_mapping = {'pyqgis_api': ('https://qgis.org/pyqgis/3.4/', 'py_qgis_api_objects.inv')}
```

Comment:
Should this be master and updated at the release branch time?
```suggestion
intersphinx_mapping = {'pyqgis_api': ('https://qgis.org/pyqgis/master/', 'py_qgis_api_objects.inv')}
```

Reply:
I think we should use 3.4, especially because the 3.4 API is released and that is our current milestone. We just need to set it to master once the documentation is released 3.4, and the new milestone version API is not available yet as a web page.

Besides, the changes you made is not enough, we need to update the `py_qgis_api_objects.inv` file, which for master may be a moving target...